Purchases/Public/RCPurchases.m

```diff
@@ -625,6 +630,15 @@ - (void)restoreTransactionsWithCompletionBlock:(nullable RCReceivePurchaserInfoB
             CALL_IF_SET_ON_MAIN_THREAD(completion, nil, [RCPurchasesErrorUtils missingReceiptFileError]);
             return;
         }
+
+        RCPurchaserInfo * _Nullable cachedPurchaserInfo = [self readPurchaserInfoFromCache];
```

Comment:
we'll move a lot of stuff out of this class eventually, but it's out-of-scope for this PR